packages/components/src/components/carousel/carousel.tsx

```diff
@@ -42,8 +42,8 @@ import {
   warnIfDeprecatedPropIsUsed,
   warnIfDeprecatedPropValueIsUsed,
 } from '../../utils';
-import { carouselTransitionDuration, getComponentCss } from './carousel-styles';
-import { gridGap } from '@porsche-design-system/utilities-v2';
+import { getComponentCss } from './carousel-styles';
+import { gridGap, motionDurationModerate, motionEasingBase } from '@porsche-design-system/utilities-v2';
```

Comment:
Now we lost the connection between styles and slider options which could lead to differing values in the future.